### PR TITLE
add: UIColors to Colors under Frontend Section

### PR DESCRIPTION
**Pull request by haseebyousuf**
## Fixes Issue

Closes: #2003

## Changes proposed

Added UiColors to Colors section under Frontend Section

## Note to reviewers

I am open to any suggestions regarding this PR.

### CODE REVIEW

**Review by Anmol-Baranwal — DISMISSED** [review]
🚀 Looks good to me.

File: database/frontend/colors.json [hunk 1]
```diff
@@ -201,5 +201,12 @@
     "url": "https://www.realtimecolors.com/",
     "category": "frontend",
     "subcategory": "colors"
+  },
+  {
+    "name": "UIColors",
+    "description": "UIColors is a interactive Tailwind CSS Color Palette UI with real examples. Create your own palette or edit anything from the library. It also allows you to import the palette directly to Figma file and more.",
```
Comment (CBID2) [inline database/frontend/colors.json:207]:
```suggestion
    "description": "UIColors is a interactive Tailwind CSS color palette UI with real examples. Create your own palette or edit anything from the library. It also allows you to import the palette directly to Figma file and more.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @haseebyousuf! :) Just had a few typos

**Review by CBID2 — APPROVED** [review]
Looks good to me @haseebyousuf! :) @Anmol-Baranwal and @rupali-codes, you're up! :)

**Review by rupali-codes — APPROVED** [review]
Lgtm

**Pull request merged**